packages/components/pie-button/package.json

```diff
@@ -7,8 +7,13 @@
   "scripts": {
     "build": "npx tsc && run -T vite build",
     "dev": "run -T vite",
+    "watch": "concurrently \"run -T vite build --watch\" \"tsup --watch\"",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "JustEatTakeaway - Design System Web Team",
-  "license": "Apache-2.0"
+  "license": "Apache-2.0",
+  "dependencies": {
+    "concurrently": "^7.6.0",
+    "tsup": "^6.6.3"
```

Comment:
We probably want to remove the `^` and pin the versions. I imagine we'd want both of these dependencies at the root of the monorepo too as we'd need them for other components?